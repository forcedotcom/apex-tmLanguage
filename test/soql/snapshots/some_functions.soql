
SELECT city_c, State_c, COUNT(Employee_Name__C) Counts
 FROM Employee__C
 GROUP BY ROLLUP (City__c, State__C)
