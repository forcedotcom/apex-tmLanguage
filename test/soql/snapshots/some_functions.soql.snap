>
>SELECT city_c, State_c, COUNT(Employee_Name__C) Counts
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^^^ source.soql keyword.query.field.apex
#             ^ source.soql punctuation.separator.comma.apex
#              ^ source.soql
#               ^^^^^^^ source.soql keyword.query.field.apex
#                      ^ source.soql punctuation.separator.comma.apex
#                       ^ source.soql
#                        ^^^^^ source.soql support.function.query.apex
#                             ^ source.soql punctuation.parenthesis.open.apex
#                              ^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                              ^ source.soql punctuation.parenthesis.close.apex
#                                               ^ source.soql
#                                                ^^^^^^ source.soql keyword.query.field.apex
> FROM Employee__C
#^ source.soql
# ^^^^ source.soql keyword.operator.query.from.apex
#     ^ source.soql
#      ^^^^^^^^^^^ source.soql storage.type.apex
> GROUP BY ROLLUP (City__c, State__C)
#^ source.soql
# ^^^^^^^^^^^^^^^ source.soql support.function.query.apex
#                ^ source.soql
#                 ^ source.soql punctuation.parenthesis.open.apex
#                  ^^^^^^^ source.soql keyword.query.field.apex
#                         ^ source.soql punctuation.separator.comma.apex
#                          ^ source.soql
#                           ^^^^^^^^ source.soql keyword.query.field.apex
#                                   ^ source.soql punctuation.parenthesis.close.apex
>